src/components/Reports/GDPR/SubscriberGdpr.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import { FormattedMessage, useIntl, FormattedDate } from 'react-intl';
+import { Helmet } from 'react-helmet';
+import { getSubscriberStatusCssClassName, extractParameter } from './../../../utils';
+import queryString from 'query-string';
+import { InjectAppServices } from '../../../services/pure-di';
+import { Loading } from '../../Loading/Loading';
+import { StarsScore } from '../../shared/StarsScore/StarsScore';
+import SafeRedirect from '../../SafeRedirect';
+
+const SubscriberGdpr = ({ location, dependencies: { dopplerApiClient } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const [state, setState] = useState({ loading: true });
+
+  const replaceSpaceWithSigns = (url) => {
```

Comment:
maybe can add in utils, what do you think?

Reply:
Fixed!